test/uk/gov/hmrc/forexrates/controllers/ForexRatesControllerSpec.scala

```diff
@@ -69,5 +69,28 @@ class ForexRatesControllerSpec extends SpecBase {
       }
     }
   }
+
+  ".get date range" - {
+    val dateTo = requestDate.plusDays(5)
+    lazy val request = FakeRequest(GET, routes.ForexRatesController.getRatesInDateRange(requestDate, dateTo, baseCurrency, targetCurrency).url)
+
+    "must return forex rates when data is found" in {
```

Comment:
Can we add a tests for failures?
Repo failures 
Json parsing failure (not sure if this one would be possible tbh)

Reply:
Could you give an example of the failure we'd want to test? We've got scenarios for the get methods where there is or isn't data found, not sure what else we should be checking. We looked at our other backends specs and it seems that's usually all we test.

Reply:
Very broadly, something like this:
`when(mockRepository.get(requestDate, dateTo, baseCurrency, targetCurrency)) thenReturn Future.failed(new Exception("couldn't connect to db"))`
What would you expect the controller to return?